doctr/models/_utils.py

```diff
@@ -161,3 +161,36 @@ def get_language(text: str) -> Tuple[str, float]:
     if len(text) <= 1 or (len(text) <= 5 and lang.prob <= 0.2):
         return "unknown", 0.0
     return lang.lang, lang.prob
+
```

Comment:
Let's add a test on `invert_list_dict_to_dict_list` and `invert_dict_list_to_list_dict`